pkg/receiver/smartagentreceiver/testdata/invalid_config.yaml

```diff
@@ -1,7 +1,3 @@
 smartagent/negativeintervalseconds:
   type: collectd/redis
   intervalSeconds: -234
-smartagent/missingrequired:
-  type: collectd/consul
-  port: 5309
-
```

Comment:
Shall we replace the `missingrequired` test config with another monitor to keep the intended test, ie: missing required settings?

Reply:
22533029f7afc7599b78d5bdcf09be2ff7ab57e1